flow_transferFiles2Apollo/createOrganism-TF.cwl

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env cwl-runner
```

Comment:
Can you just use createOrganism.cwl in the apolloServer-workflow.cwl? That would be fewer files to maintain.

Reply:
I created createOrganism-TF.cwl because I need to do some modification for line 36.
Because apolloServer-workflow doesn't contain flow-apollo_data_processing, it dosen't have the 2bi output. 
Therefore, I can't use 2bi as an input for line 36. That't why I created a new file and modified line 36 to "/*2bi" .

Reply:
Ah, got it. Thanks!